doc/dev/cable_cell.rst

```diff
@@ -80,7 +80,7 @@ Discretisation splits the segments described by the morphology into *control
 volumes* (CV; sometimes called *compartments*) according to a ``cv_policy``.
 This allows us to construct a system of linear equations, the Hines matrix, to
 describe the evolution of the CV voltages according to the cable equation. Refer
-to :ref:`Discretisation <discretisation>` and :ref:`Cable equation
+to the :ref:`decor's discretization method <pycablecell-decor>` and :ref:`Cable equation
```

Comment:
Don't let Sam see this ;)

Reply:
I can leave the dead link as I did with the above links - is the chapter "Discretization" something we intend to write down at some point?

Reply:
A minor war was fought over discreti**s**ation vs discreti**z**ation. Don't ask :) No request for change meant.